content/metadata-log/index.md

```diff
@@ -180,7 +176,7 @@ a0 =
   ]
 ```
 
-Note a delta is an ordered set ordered by key.
+Note that a delta is a data set ordered by key.
```

Comment:
Let's talk about this one. A "Set" is unordered by definition. An "Ordered Set" is a different data structure with different expectations. When I say "is an ordered set ordered by key" I mean:

1. A delta is an "ordered set" (as opposed as "a set")
2. The "ordered set" ordering criteria is by the element key.